contracts/cw721-remarkables/examples/schema.rs

```diff
@@ -0,0 +1,25 @@
+use std::env::current_dir;
+use std::fs::create_dir_all;
+
+use cosmwasm_schema::{export_schema_with_title, remove_schemas, schema_for};
+
+use cw721::{AllNftInfoResponse, NftInfoResponse};
+use cw721_remarkables::Metadata;
+
+fn main() {
+    let mut out_dir = current_dir().unwrap();
+    out_dir.push("schema");
+    create_dir_all(&out_dir).unwrap();
+    remove_schemas(&out_dir).unwrap();
+
+    export_schema_with_title(
+        &schema_for!(AllNftInfoResponse<Metadata>),
+        &out_dir,
+        "AllNftInfoResponse",
+    );
+    export_schema_with_title(
+        &schema_for!(NftInfoResponse<Metadata>),
+        &out_dir,
+        "NftInfoResponse",
+    );
```

Comment:
Since this cw721 contract only has the difference about Metadata from cw721-base, we just need to expose `AllNftInfoResponse<Metadata>` and `NftInfoResponse<Metadata>`. The rest can be obtained from cw721-base.